client/jetpack-cloud/sections/partner-portal/license-multi-product-card/index.tsx

```diff
@@ -85,7 +85,8 @@ export default function LicenseMultiProductCard( props: Props ) {
 				onSelect();
 			}
 		}
-	}, [ onSelect, product.slug, suggestedProduct ] );
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can we add a comment above as to why we are disabling this? 

Very high chance that this will be removed in the future.

```suggestion
        //  Do not add onSelect to the dependency array as it will cause an infinite loop
		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Reply:
Totally agree. 👍